data/human/human missions.txt

```diff
@@ -5547,3 +5547,130 @@ mission "Quicksilver Mixup 1"
+			label turn
+			`	Both of you turn your heads towards the <npc> to see what's happening, but as you do you hear the deafening blare of warning sirens through <origin>'s spaceport; at the same time, the <npc> suddenly jolts up with its loading ramp still extended.`
+				goto hijack
+			
+			label ignore
+			`	You turn back towards the <ship> only to hear the deafening blares of warning sirens through <origin>'s spaceport. Looking around instinctively, you notice from the corner of your eye that the <npc> suddenly jolts up with its loading ramp still extended.`
+
+			label hijack
+			`	"Hey, what's going on here?" the officer tries to shout over the noise of the sirens and the repulsors, but the rogue <npc model> veers rapidly and uncertainly towards his direction and the <ship>. He can barely jump down for cover before the ship clips a stack of cargo crates, sending them flying into the refueling platform. With its repulsors at full power, it narrowly avoids colliding with the <ship> and flees away into the sky.`
+			`	The Syndicated Security officer, dazed and coated in dust kicked up from the unpermitted departure, only just pulls himself up before another person runs up to accost him. "Oi, those kids, they just hijacked my <npc model>! Get someone on their tail!"`
+			choice
+				`	"I'll do it!"`
+				`	(Let Syndicated Security deal with it.)`
+					goto securityjob
+			`	The captain of the stolen <npc model> turns towards you. "Thank you, Captain! Don't destroy the <npc>, I'm already late for my bloody contract. Agh, this doesn't help at all," he mutters to himself.`
+			`	The officer looks back at you, having dusted himself off and regained his senses. "Yes, go and deal with his ship now. Don't board it, we'll have Syndicated Security ships come after you to board the hijacked ship and tug it down. Go!"`
+				launch
+			
+			label securityjob
+			`	The officer, slightly flustered, calls for a small squadron of Syndicated Security ships to chase the hijacked <npc>, then runs off to one of <origin>'s spaceport terminals to file the paperwork with the captain.`
+				decline
+	
+	npc save disable
+		personality launching daring uninterested target
+			confusion 40
+		government "Bounty (Disguised)"
+		ship "Argosy (Blaster)" "Moonshake Express"
+		dialog `The <npc> has been disabled. Syndicated Security will now deal with the hijackers, but you should go back and report to the captain whose ship was hijacked.`
+		on board
+			set "boarded hijacked argosy"
+			dialog `You have failed this mission because you boarded the ship, against the orders of the Syndicated Security officer. The captain whose ship was hijacked won't be impressed.`
+			fail
+
+	on fail
+		conversation
+			`When you land, the captain of the <npc> comes up to you, his face rather red.`
+			`	He grabs your shoulders and shakes you hard, shouting, "Didn't I tell you not to <argosy hijacking variable> my ship? Do you know how many credits I've just lost because of you?"`
```

Comment:
It looks like substitutions are set in stone once the mission is accepted, so this dialog doesn't update properly. Might just want to make a `branch` here instead.

![image](https://user-images.githubusercontent.com/4471575/228082681-4d6137e3-9e3c-48ad-99ad-a7d37577df34.png)